<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWindow</class>
 <widget class="QMainWindow" name="RoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image: url(:/image/room/房间主背景.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Ledw_1" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QWidget" name="Ledw_2" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="Fanw" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closefan_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="Doorw" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closedoor_ico.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Led_1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>113</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Led_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Fan">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Door">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
